src/main/java/com.company/linkedlist/ReverseBetween.java

```diff
@@ -0,0 +1,28 @@
+package com.company.linkedlist;
+
+public class ReverseBetween {
+    public ListNode reverseBetween(ListNode head, int m, int n) {
+        if (head == null || m == n) {
+            return head;
+        }
+        ListNode dummy = new ListNode(0);
+        dummy.next = head;
+        ListNode prev = dummy;
+        for (int i = 0; i < m; i++) {
+            prev = prev.next;
+        }
+        ListNode current = prev.next;
+        ListNode tail = current;
+        for(int i = m; i <= n; i++) {
+            current.next = prev.next;
```

Comment:
prev.next is current. 
How come again current.next point to prev.next?

Did you try it with an example?
This won't work!

Reply:
fixed and checked in leetcode